boilerplate/app/models/root-store/setup-root-store.ts

```diff
@@ -1,9 +1,9 @@
 import { onSnapshot } from "mobx-state-tree"
 import { RootStoreModel, RootStore } from "./root-store"
-import { Environment } from "./environment"
-import * as storage from "../lib/storage"
-import { Reactotron } from "../services/reactotron"
-import { Api } from "../services/api"
+import { Environment } from "../environment"
+import * as storage from "../..//utils/storage"
```

Comment:
two slashes typo

Reply:
@mangei Would you mind sending in a PR?

Reply:
@jamonholmgren https://github.com/infinitered/ignite-bowser/pull/160

Reply:
I just read this. Thanks @dashracer for fixing it! :)